### PR TITLE
Migrates tabs from jsadmin-v2

### PR DESCRIPTION
**Pull request by sbelzile-Coveo**

### CODE REVIEW

File: docs/_components/components/17_tab.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <div class="mod-header-padding mod-form-top-bottom-padding">
+            Content of Tab 1
+        </div>
+    </div>
+    <div class="tab-pane">
+        <div class="mod-header-padding mod-form-top-bottom-padding">
+            Content of Tab 2
+        </div>
+    </div>
+</div>
```
Comment (GermainBergeron) [inline docs/_components/components/17_tab.html]: You can add a small script under the div to change the active tab/pane, it's nice when we can interact with the components

File: scss/components/tab.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    visibility: visible;
+
+    @extend .slideInLeft;
+  }
+
+  .loading, & > .spinner {
+    margin-top: 50px;
+  }
+}
```
Comment (GermainBergeron) [inline scss/components/tab.scss]: I guess this is taken directly from v2 without changes?

Comment (sbelzile-Coveo) [inline scss/components/tab.scss]: exact

File: scss/animations.scss [hunk 3]
```diff
@@ -46,3 +46,24 @@
...
+    -webkit-transform: translateX(0);
+    -ms-transform: translateX(0);
+    transform: translateX(0);
+  }
+}
+
+.slideInLeft {
+  -webkit-animation-name: slideInLeft;
+  animation-name: slideInLeft;
+}
```
Comment (wfortin) [inline scss/animations.scss]: Remove the -webkit, -ms, prefixes. Autoprefixer takes care of that.

File: scss/components/tab.scss [hunk 4]
```diff
@@ -0,0 +1,73 @@
+.tab-navigation {
+  min-height: 36px;
+  padding: 0 $header-padding;
+
+  border-bottom: 1px solid $medium-grey;
```
Comment (pochretien) [inline scss/components/tab.scss]: you can use $default-border

File: scss/components/tab.scss [hunk 5]
```diff
@@ -0,0 +1,73 @@
+.tab-navigation {
+  min-height: 36px;
```
Comment (JPLachance) [inline scss/components/tab.scss]: We prefer defining and using variables for all important properties like the height, the font size, margins, etc. See the variables.scss file.

**Pull request merged**